sshd-core/src/test/java/org/apache/sshd/certificates/GenerateOpenSSHClientCertificateTest.java

```diff
@@ -154,7 +150,12 @@ public void signCertificate() throws Exception {
                 .publicKey(clientPublicKey)
                 .id("user01")
                 .principals(Collections.singletonList("user01"))
+                .criticalOptions(Arrays.asList(
+                        new OpenSshCertificate.CertificateOption("force-command", "/path/to/script.sh"),
+                        new OpenSshCertificate.CertificateOption("source-address", "127.0.0.1/32"),
+                        new OpenSshCertificate.CertificateOption("verify-required")))
```

Comment:
Please also add critical options in the `GenerateOpenSshClientCertificateOracleTest`. That one tests against pre-existing certificates generated using OpenSSH.

Reply:
@tomaswolf i didn't add there as would be required to change all existing certs as message is now different meaning signature and message check will not pass.
Probably better would be to add separate test for these. I will see later what would be best and will update

Reply:
Just regenerate the certs with OpenSSH and replace the files in test/resources. The Java code shows how they would need to be regenerated to match.

Reply:
ok, just regenerated certs but i have separated them from the old ones as there is other test which is using same and it fails with dummy critical options (source-address, force-command). There is sh script which can be used to easily generate/regenerate if needed